### DISCUSSION TITLE
文章列表按时间进行排序

### DISCUSSION THREAD

**Discussion by vic2ray** [Qexo/Qexo#133] [Ideas]
目前的文章列表似乎是按照A-Z的顺序排列，也没有找到可设置的地方。

希望以时间倒序排序，能够方便查看和编辑近期文章。Much thanks!

**Comment by am-abudu** [comment]
Github并无相关API 故无法实现

**Reply by 37748846** [reply]
，对编辑或者发布的文章直接读取创作时间和更改时间。可以通过数据库或静态页面生成对应的时间戳，应该可以实现双排序吧？

**Reply by am-abudu** [reply]
> ，对编辑或者发布的文章直接读取创作时间和更改时间。可以通过数据库或静态页面生成对应的时间戳，应该可以实现双排序吧？

全部读取一遍需要大量的时间生成索引，严重超出Serverless程序的限制

**Comment by God-2077** [comment]
我不太懂Serverless程序，但我想有些人是在自家服务器搭建Qexo的，应该不受Serverless限制吧？可以在设置里弄个开启时间排序吧？

~~我现在用的是 Decap CMS ，基于 Netlify Identity 和 Git 网关的，可以实现基于时间进行排序。~~

**Reply by am-abudu** [reply]
这是可能实现的 但是目前只能为本地用户提供支持（github/gitlab无法支持）

**Reply by am-abudu** [reply]
此处我提到的平台是博客文件的存储平台（见hexoweb/libs/platforms）

**Comment by mcxiaochenn** [comment]
个人比较喜欢用``2025/09/1.name.md``
也就是年/月/本月的第几篇.文章主题.md
导致了一个问题就是每次翻新文章要从后往前翻而非从前往后，能不能加个文件名排序a-z z-a修改的地方
